hack/deploy-test-cluster.sh

```diff
@@ -6,15 +6,27 @@ set -ex
 docker run -d -p 5000:5000 --rm --network kubeadm-dind-net --name registry registry:2
 kubectl config view --raw > ./cluster/dind-cluster/config
 
-# Deploy CNA
-./cluster/dind-cluster/kubectl create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/namespace.yaml
-./cluster/dind-cluster/kubectl create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/network-addons-config.crd.yaml
-./cluster/dind-cluster/kubectl create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/operator.yaml
-./cluster/dind-cluster/kubectl create -f ./hack/cna/cna-cr.yaml
-
-# Deploy Kubevirt
-./cluster/dind-cluster/kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/v0.20.4/kubevirt-operator.yaml
-./cluster/dind-cluster/kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/v0.20.4/kubevirt-cr.yaml
+
+echo 'Deploying Linux bridge CNI and Multus ...'
```

Comment:
Please note that this will go away as soon as we move to prow (we will drop dind)